client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -185,6 +179,10 @@ export default {
     },
 
     updateRuleConstraint (rule) {
+      if (!rule.constraints) {
```

Comment:
Since there were changes here, make sure the unique values works as expected.
Mention it to Bojan that this should also be checked when QCing